src/components/Header/Header.tsx

```diff
@@ -0,0 +1,79 @@
+import { FunctionComponent, useState, useEffect } from 'react';
+import Logo from '../../assets/icons/Logo.png';
+import Burger from '../../assets/icons/burger-icon.svg';
+import BurgerClosed from '../../assets/icons/burger-icon-closed.svg';
+import NavLinks from './NavLinks/NavLinks';
+
+import styles from './Header.module.scss';
+
+interface HeaderProps { }
```

Comment:
Po co to

Reply:
It was in Wojteks version of header so i teft it there